userportaldatamodel/associated_user.py

```diff
@@ -25,12 +25,13 @@ class AssociatedUserRoles(Base):
 
     create_date = Column(DateTime(timezone=False), server_default=func.now())
     update_date = Column(DateTime(timezone=False), server_default=func.now(), onupdate=func.now())
-   
+    
+    projects = relationship('ProjectAssociatedUser', backref='role')
```

Comment:
could you move this to the ProjectAssociatedUser class before or after to the role_id in line 86 and swap its attribute name and backref. Just to be consistent with the other code and keep all the relationships in the same place?

Reply:
update is pushed